### PR TITLE
feature: Navigation breadcrumbs

### PR DESCRIPTION
**Pull request by IainSAP**
Implements https://github.com/SAP/yeoman-ui/issues/743

- Adds optional answer history (breadcrumbs) to the navigation steps.
- Adds tests
- Updates styling
- Adds types to enable code-assist for frontend tests

### CODE REVIEW

File: packages/generator-foodq/package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "generator-foodq",
   "version": "1.10.5",
   "private": true,
-  "description": "A sample generator to preview all the options available when using the Yeoman-UI",
+  "description": "A sample generator to preview all the options available when using the Yeoman-UI (updated)",
```
Comment (EstherPerelman) [inline packages/generator-foodq/package.json:5]: I didn't get why need to update the description?

**Review by EstherPerelman — APPROVED** [review]
The code looks good to me!
@IainSAP I still would like to get confirmation about the font from the ux because it looks a bit unclear

**Comment by EstherPerelman** [comment]
@IainSAP  I still would like to get confirmation about the font from the UX because it looks a bit unclear:
image

**Comment by IainSAP** [comment]
> @IainSAP I still would like to get confirmation about the font from the UX because it looks a bit unclear: 

It  looks ok for me and is using a vscode variable for the font, is it ok to merge? :
image

File: packages/generator-foodq/package.json (hunk 1 above)
Comment (IainSAP) [reply packages/generator-foodq/package.json:5]: Thats a mistake I can revert

**Comment by EstherPerelman** [comment]
> > @IainSAP I still would like to get confirmation about the font from the UX because it looks a bit unclear:
> 
> Thanks @EstherPerelman . It looks ok for me and is using a vscode variable for the font, is it ok to merge? :
> 
> image

@IainSAP I noticed that it only appears unclear on my external screen so probably it is related to its bad resolution and not to the code.. sorry for that, 
Besides, it might be better to add a test for an empty string:
image

**Comment by IainSAP** [comment]
> > > @IainSAP I still would like to get confirmation about the font from the UX because it looks a bit unclear:
> > 
> > 
> > Thanks @EstherPerelman . It looks ok for me and is using a vscode variable for the font, is it ok to merge? :
> 
> @IainSAP I noticed that it only appears unclear on my external screen so probably it is related to its bad resolution and not to the code.. sorry for that, Besides, it might be better to add a test for empty string case: 
...

Done thx!

**Comment by EstherPerelman** [comment]
@IainSAP great, I guess you are able to merge

**Comment by IainSAP** [comment]
Thx @EstherPerelman
